packages/spacecat-shared-aa-api-client/src/rumtoaamapper.js

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright 2024 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+import { EVARS_KEY, EVENTS_KEY } from './helpers.js';
+
+export default class RUMToAAMapper {
```

Comment:
I think the mapper belongs to the importer lambada I would remove the class from here